app/views/state_file/questions/nj_homeowner_eligibility/edit.html.erb

```diff
@@ -0,0 +1,29 @@
+<%
+    title = t(".title", filing_year: @filing_year)
+    content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
+    <h1 class="h2"><%= title %></h1>
+    <p><%= t(".label") %></p>
+
+    <div class="tight-checkboxes spacing-above-0 question-with-follow-up">
+      <%= f.cfa_checkbox(:homeowner_home_subject_to_property_taxes, t(".homeowner_home_subject_to_property_taxes"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:homeowner_more_than_one_main_home_in_nj, t(".homeowner_more_than_one_main_home_in_nj"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:homeowner_shared_ownership_not_spouse, t(".homeowner_shared_ownership_not_spouse"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <div class="question-with-follow-up__question">
+        <%= f.cfa_checkbox(:homeowner_main_home_multi_unit, t(".homeowner_main_home_multi_unit"), options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#homeowner_multi_unit_followup"}) %>
+      </div>
+      <div class="question-with-follow-up__follow-up" id="homeowner_multi_unit_followup">
```

Comment:
[pebble] This will need some ARIA markup to meet [WCAG 4.1.2, which mandates that users be notified of changes](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).

[The UK digital services team has put a lot of care and research into this topic.](https://accessibility.blog.gov.uk/2021/09/21/an-update-on-the-accessibility-of-conditionally-revealed-questions/) The [TLDR in their component lib seems to be](https://design-system.service.gov.uk/components/radios/) that associating one input is fine (though I'm a little skeptical of our choice to not visually differentiate it) and we should use `aria-controls`.

Reply:
[dust/pebble] It looks like gov UK is also using aria-expanded on their example, and voiceover announced whether or not something was expanded when I tried it on their example. So it might be worth looking into adding that too, or figuring out whether we need that

Reply:
There exists `cfa_checkbox_set` but it's not being used in the codebase, and doesn't support HTML options (including aria-expanded but also just the ones we need to get the checkbox values and the expand/collapse function working). I think we should give this feedback to CFA but it's not part of this codebase, it's a library being pulled in.

Reply:
Maybe we should use checkbox set for now (assuming it outputs the correct html) and ask them to support aria-expanded? And if that's not happening, perhaps consider a design change?

Reply:
@mmazanec22 Sorry I didn't explain correctly - that was my plan, but the checkbox set doesn't appear to support the expand/collapse function. Like, I can't give a particular checkbox a particular class/data tag which is what allows one checkbox to trigger the other one appearing. So at least with the current way the set works, I don't think we can use it in this page because it can't support the appear/disappear functionality